src/ess/imaging/workflow.py

```diff
@@ -91,18 +99,15 @@ def YmirImageNormalizationWorkflow() -> sl.Pipeline:
 
         \\text{where } i \\text{ is an index of an image.}
 
-        \\text{Pixel values less than sample_threshold}
-
-        \\text{ are replaced with sample_threshold}.
+        \\text{Pixel values less than sample_threshold are masked.}
 
     .. math::
 
         BackgroundImage = mean(OpenBeam, dim=\\text{'time'})
         - mean(DarkCurrent, dim=\\text{'time'})
 
-        \\text{Pixel values less than } \\text{background_threshold}
+        \\text{Pixel values less than background_threshold are masked}
```

Comment:
```suggestion
        \\text{Pixel values less than background\_threshold are masked}
```
I can't remember if we need to escape underscores in latex...

Reply:
It's wrapped in \\text so it's fine : D